sass/_theme_rst.sass

```diff
@@ -174,6 +174,11 @@
       vertical-align: top
     code
       color: $gray
+  .wy-table-responsive.citation, .wy-table-responsive.footnote
```

Comment:
Add a comment about what this is working around.

Reply:
Done (without mentioning GitHub issue)